styles/styles.scss

```diff
@@ -59,3 +59,4 @@
 // Theme partials
 @import '../Magento_Cms/styles/module';
 @import '../Magento_Theme/styles/module';
+@import '../Magento_Contact/styles/module';
```

Comment:
we do not import components style in magento module styles in theme. There should be only styles needed in theme to implement components (some layout customisation) but only if it is **really** necessary